kafka-connect/README.md

```diff
@@ -0,0 +1,55 @@
+# Kafka Connect Resources
+
+This directory contains the OpenShift templates and Containerfile needed to build and deploy a KafkaConnect cluster using the Streams for Apache Kafka operator from Red Hat
+
+### Kafka Connect Image
+
+The included Containerfile can be used to build the Kafka Connect image needed to spin up a Kafka Connect cluster using Streams for Apache Kafka. It can be built using podman or docker and can also be built via AppSRE/Konflux image build processes. Currently, the Connect image only contains the Debezium connector for PostgreSQL, any other connectors or plugins/libs required would need to be added in your own fork of the Containerfile.
+
+Plugins are installed using the [docker-maven-download](https://github.com/debezium/container-images/blob/main/connect-base/2.7/docker-maven-download.sh) script provided by Debezium's container-images repo and is useful for installing other plugins and libs. Review the script for more information on how to use it.
+
+To build the image locally:
+
+`podman build -t quay.io/<YOUR-REPOSITORY>/<SERVICE_NAME>-kafka-connect -f Containerfile`
+
+
+### Kafka Connect Deployment
+The KafkaConnect CR templates can be used to deploy a Kafka Connect cluster using the image built with the provided Containerfile. The template is designed to allow for multiple service providers to use the same template while avoiding name duplication as these Connect clusters will likely live in the same namespace
+
+There are 2 versions of the KafkaConnect CR: one with authentication, one without
+
+The `kafkaconnect-no-auth.yml` template is useful for Ephemeral, where the Clowder-provided Kafka cluster can be used for the Connect cluster and does not require any authentication.
+
+The `kafkaconnect-w-auth.yml` use a useful template example for adding to your deployment file for targeting stage/production environments where MSK is likely used and is configured with SASL/SCRAM. In order to authenticate, you will need a Kakfa user configured for the MSK cluster. See the **Managed Streaming for Apache Kafka (MSK) via App-Interface** section of the App Interface docs on how to add users.
```

Comment:
```suggestion
The `kafkaconnect-w-auth.yml` template is useful for adding to your deployment file for targeting stage/production environments where MSK is likely used and is configured with SASL/SCRAM. In order to authenticate, you will need a Kakfa user configured for the MSK cluster. See the **Managed Streaming for Apache Kafka (MSK) via App-Interface** section of the App Interface docs on how to add users.
```
I was just a bit confused on the wording used here. I think aligning it with similar wording as the no-auth example makes sense.